GOVERNANCE.md

```diff
@@ -0,0 +1,159 @@
+
+In all cases, all and only [team members](#team-members) are eligible to vote, with the sole exception of the forced removal of a team member, in which said member is not eligible to vote.
+
+Discussion and votes on personal matters (including but not limited to team membership and maintainership) are held in private on the [team mailing list][team]. All other discussion and votes are held in public on the [developer mailing list][team].
+
+For public discussions, anyone interested is encouraged to participate. Formal power to object or vote is limited to [team members](#team-members).
+
+### Consensus
+
+The default decision making mechanism for the Prometheus Project is [lazy consensus][lazy]. This means that any decision on technical issues is considered supported by the [team][team] as long as nobody objects.
+
+Silence on any consensus decision is implicit agreement and equivalent to explicit agreement. Explicit agreement may be stated at will. Decisions may, but do not need to be called out and put up for decision on the [developers mailing list][devs] at any time and by anyone.
+
+Consensus decisions can never override or go against the spirit of an earlier explicit vote.
+
+If any [team member](#team-members) raises objections, we work together towards a solution that all involved can accept. This solution is again subject to lazy consensus.
+
+In case no consensus can be found, but a decision one way or the other must be made, any [team member](#team-members) may call a formal [majority vote](#majority-vote).
+
+### Majority vote
+
+Majority votes must be called explicitly in a separate thread on the appropriate mailing list. The subject must be prefixed with `[VOTE]`. In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
+
+Voting must be open for at least 1 week (7 days from the beginning of the vote). The end date should be clearly stated in the call to vote.
+
+Votes may take the form of a single proposal, with the option to vote yes or no; or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if more vote in favour than against.
+
+A vote on multiple alternatives is considered decided in favour of one alternative if any alternative receives more than half of the total votes cast. Members may vote against all alternatives by voting "no". Should no alternative reach a majority, another vote on a reduced number of options may be called separately.
+
+### Supermajority vote
+
+Supermajority votes must be called explicitly in a separate thread on the appropriate mailing list. The subject must be prefixed with `[VOTE]`. In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
+
+Voting must be open for at least 1 week (7 days from the beginning of the vote). The end date should be clearly stated in the call to vote.
```

Comment:
end date and previous-discussion-reference are "should" to avoid invalidating a whole vote due to a minor form error (or maybe even there being another thread of discussion that the person calling the vote did not know about). it's a strong should but not an absolute requirement.